### DISCUSSION TITLE
Remove the left-side navigation panel

### DISCUSSION THREAD

**Discussion by xhqiao89** [tethysplatform/tethys#1131] [Q&A] [answered]
![app1](https://github.com/user-attachments/assets/b3a19974-4730-4d05-b50c-cec727a752bc)
Hi there, we are developing a web app using Tethys. But we don't know how to remove the left-side navigation panel (the grey part).  Could someone guide us on how to do this? Thank you so much!

**Comment by swainn** [comment, marked as answer]
Hi @xhqiao89 probably the easiest way to do that would be to use a different base template that doesn't have the left side navigation. Take a look at this overview of Base Templates and the list of Additional Base Templates: https://docs.tethysplatform.org/en/stable/tethys_sdk/templating.html#base-templates and https://docs.tethysplatform.org/en/stable/tethys_sdk/templating.html#additional-base-templates

To use one of the base templates, replace the ``extends`` block in your template with something like this:

```
{% extends "tethys_apps/<base_template_name>.html" %}
```

**Reply by xhqiao89** [reply]
Thank you so much!
